src/resources/js/views/multiday-events.js

```diff
@@ -9,7 +9,7 @@ tribe.events = tribe.events || {};
 tribe.events.views = tribe.events.views || {};
 
 /**
- * Configures Views Object in the Global Tribe variable
+ * Configures Multiday Events Object in the Global Tribe variable
```

Comment:
Are we planning to handle multi-day events for the week view here as well? If not it'd be ideal to specify the file name (obj, etc) for the month -> e.g.: `views/month-multiday-events.js`

Reply:
good call, will make that change. if we find that week view can basically reuse most/all of the js, we can change this back then.